### PR TITLE
feature: invoke keyman compiler

### LINKED ISSUE #19: Run the compiler within the Web Worker
Simply invoke the Keyman Lexical model compiler and produce JavaScript output.

### PR DESCRIPTION
**Pull request by eddieantonio**
Invokes the Keyman compiler from the tests, but should invoke it from the worker soon 👍 

Resolves #19

### CODE REVIEW

File: test/test-generate-wordlist.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        ["TŦE", 13644],
+        ["E", 9134],
+        ["SEN", 4816],
+      ]),
+    ],
+  });
+  // it will compile to a JavaScript function!
+  t.regex(code, /\bfunction\b/);
+
+  t.notThrows(function () {
```
Comment (IanSeng) [inline test/test-generate-wordlist.ts]: **suggestion** can we use arrow function ?

File: test/test-generate-wordlist.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    // We should be able to eval that function and get its code:
+    new Function(code);
```
Comment (IanSeng) [inline test/test-generate-wordlist.ts:31]: same as above

**Review by IanSeng — CHANGES_REQUESTED** [review]

File: test/test-generate-wordlist.ts (hunk 2 above)
Comment (eddieantonio) [reply test/test-generate-wordlist.ts:31]: ...not for this one. The purpose of this is to **parse the JavaScript code** generated by the lexical model compiler. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/Function

File: test/test-generate-wordlist.ts (hunk 1 above)
Comment (eddieantonio) [reply test/test-generate-wordlist.ts]: Yes for this one! ...but

**Review by IanSeng — APPROVED** [review]
👍

**Pull request merged**